src/forscape_parser.cpp

```diff
@@ -22,7 +22,7 @@ namespace Forscape {
 namespace Code {
 
 Parser::Parser(const Scanner& scanner, Typeset::Model* model) noexcept
-    : tokens(scanner.tokens), errors(model->errors), model(model), error_node(NONE) {}
+    : tokens(scanner.tokens), error_stream(Program::instance()->error_stream), model(model), error_node(NONE) {}
```

Comment:
Using the Program for error reporting is bad here because Models should be fully independent until after lexical symbol resolution.

Reply:
Models have their own error stream buffers now